src/testcases/parallel/simple.test.ts

```diff
@@ -446,7 +447,7 @@ describe('Neutron / Simple', () => {
       });
     });
 
-    describe('Failing sudo handlers', () => {
+    describe('Not enough amount of tokens on contract to pay fee', () => {
```

Comment:
looks like a change by a mistake that shouldn't have been made

```suggestion
    describe('Failing sudo handlers, () => {
```